test.py

```diff
@@ -52,6 +52,7 @@
 legate_tests = []
 legate_tests.extend(glob.glob("tests/*.py"))
 legate_tests.extend(glob.glob("examples/*.py"))
+legate_tests.extend(glob.glob("random/*.py"))
```

Comment:
I don't think there would be many random tests that would warrant a separate directory for them. Even if they do, they should go under `tests` and not be a top-level directory like this.